make docker is failing as of today

Looks like a old setuptools version is having trouble parsing complex requirements in the newest schema-salad version.

```
[snip]
Collecting cgcloud-lib==1.6.0a1.dev393 (from toil==3.12.0a1)
  Downloading cgcloud-lib-1.6.0a1.dev393.tar.gz (44kB)
Collecting futures==3.0.5 (from toil==3.12.0a1)
  Downloading futures-3.0.5-py2-none-any.whl
Collecting cwltool==1.0.20170822192924 (from toil==3.12.0a1)
  Downloading cwltool-1.0.20170822192924.tar.gz (277kB)
Collecting schema-salad<3,>=2.6 (from toil==3.12.0a1)
  Downloading schema-salad-2.6.20170927145003.tar.gz (311kB)
    Complete output from command python setup.py egg_info:
    error in schema-salad setup command: Invalid environment marker: python_version>="3"
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-8j0tUs/schema-salad/
/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
The command '/bin/sh -c pip install toil-3.12.0a1.tar.gz[aws,mesos,encryption,cwl]' returned a non-zero code: 1
Makefile:198: recipe for target 'docker' failed
make: *** [docker] Error 1
```

--- a/docker/Dockerfile.py
+++ b/docker/Dockerfile.py
@@ -77,7 +77,10 @@ print(heredoc('''
     RUN chmod 777 /usr/bin/waitForKey.sh
 
     # The stock pip is too old and can't install from sdist with extras
-    RUN pip install --upgrade pip==8.1.2
+    RUN pip install --upgrade pip==9.0.1
+
+    # Default setuptools is too old
+    RUN pip install --upgrade setuptools==36.5.0
 
     # Include virtualenv, as it is still the recommended way to deploy pipelines
     RUN pip install --upgrade virtualenv==15.0.3

